root/user/privileged.tt

```diff
@@ -1,15 +1,4 @@
-[% BLOCK results %]
-<p>
-  [% IF editors.size %]
-    [%- FOR editor=editors -%]
-      [%- link_entity(editor) -%]
-      [%- IF loop.count < editors.size -%], [%- END -%]
-    [%- END -%]
-  [% ELSE %]
-    [% l('No users found') %]
-  [% END %]
-</p>
-[% END %]
+[%- React.embed(c, 'components/UserInlineList', {editors => editors}) -%]
```

Comment:
The path must be relative from `root/`, see point 2. ii. at [Porting TT to React](https://github.com/metabrainz/musicbrainz-server/blob/master/HACKING.md#porting-tt-to-react), that is `user/components/UserInlineList` here.